### ISSUE TITLE
Help for generator is not displayed

### ISSUE THREAD

**Issue by jagin** [nrwl/nx#14062]
### Current Behavior

Runing `npx nx generate @nrwl/react:library --help` I get the following help:

```
$ npx nx generate @nrwl/react:library --help
nx generate <generator> [_..]

Generate or update source code (e.g., nx generate @nrwl/js:lib mylib).

Positionals:
  generator  Name of the generator (e.g., @nrwl/js:library, library)                                 [string] [required]

Options:
      --version      Show version number                                                                       [boolean]
  -d, --dryRun       Preview the changes without updating files                               [boolean] [default: false]
      --interactive  When false disables interactive input prompts for options                 [boolean] [default: true]
      --verbose      Prints additional information about the commands (e.g., stack traces)    [boolean] [default: false]
```

### Expected Behavior

I should get the options available for `@nrwl/react:library` generator.

### Steps to Reproduce

1. I creaed a new NX Workspace presetting a monorepo with a single React application with the following command `npx create-nx-workspace@latest hcr --preset=react-monorepo`
2. I wanted to see `@nrwl/react:library` options with command `npx nx g @nrwl/react:library --help`

### Nx Report

```shell
❯ npx nx report

 >  NX   Report complete - copy this into the issue template

   Node : 16.14.2
   OS   : darwin arm64
   npm  : 8.5.0

   nx : 15.4.2
   @nrwl/angular : Not Found
   @nrwl/cypress : 15.4.2
   @nrwl/detox : Not Found
   @nrwl/devkit : 15.4.2
   @nrwl/esbuild : Not Found
   @nrwl/eslint-plugin-nx : 15.4.2
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 15.4.2
   @nrwl/js : 15.4.2
   @nrwl/linter : 15.4.2
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : 15.0.2
   @nrwl/nx-plugin : Not Found
   @nrwl/react : 15.4.2
   @nrwl/react-native : Not Found
   @nrwl/rollup : 15.4.2
   @nrwl/schematics : Not Found
   @nrwl/storybook : Not Found
   @nrwl/web : 15.4.2
   @nrwl/webpack : 15.4.2
   @nrwl/workspace : 15.4.2
   @nrwl/vite : 15.4.2
   typescript : 4.8.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
```

**Comment by adlion** [comment]
+1

**Comment by sebastiandg7** [comment]
This is also happening when looking for `--help` with executors.

**Comment by AgentEnder** [comment]
I think this should be fixed in #14187
